.gitignore

```diff
@@ -1 +1,3 @@
 node_modules/
+coverage/
+package-lock.json
```

Comment:
Maybe we should consider actually not ignoring this file, so people will know they have it (and thus should remove it)?